src/webgpu/api/validation/image_copy/buffer_texture_copies.spec.ts

```diff
@@ -304,3 +304,43 @@ g.test('depth_stencil_format,copy_buffer_offset')
       unreachable();
     }
   });
+
+g.test('sample_count')
+  .desc(
+    `
+  Test that the texture sample count. Check that a validation error is generated if sample count is
+  not 1.
+  `
+  )
+  .params(u =>
+    u //
+      .combine('sampleCount', [1, 4])
+      .beginSubcases()
+      // writeTexture is handled by writeTexture.spec.ts.
+      .combine('copyType', ['CopyB2T', 'CopyT2B'] as const)
+  )
+  .fn(async t => {
+    const { sampleCount, copyType } = t.params;
+    const texture = t.device.createTexture({
+      size: { width: 16, height: 16 },
+      sampleCount,
+      format: 'bgra8unorm',
+      usage: GPUTextureUsage.COPY_SRC | GPUTextureUsage.COPY_DST,
+    });
+
+    const uploadBufferSize = 32;
+    const buffer = t.device.createBuffer({
+      size: uploadBufferSize,
+      usage: GPUBufferUsage.COPY_SRC | GPUBufferUsage.COPY_DST,
+    });
+
+    const textureSize = { width: 1, height: 1, depthOrArrayLayers: 1 };
+
+    const isSuccess = sampleCount === 1;
+
+    if (copyType === 'CopyB2T') {
+      t.testCopyTextureToBuffer({ texture }, { buffer }, textureSize, isSuccess);
+    } else if (copyType === 'CopyT2B') {
+      t.testCopyBufferToTexture({ buffer }, { texture }, textureSize, isSuccess);
```

Comment:
These are reversed:
```suggestion
      t.testCopyBufferToTexture({ buffer }, { texture }, textureSize, isSuccess);
    } else if (copyType === 'CopyT2B') {
      t.testCopyTextureToBuffer({ texture }, { buffer }, textureSize, isSuccess);
```

Reply:
Oops, sorry for the stupid mistake. Fixed.